src/libclipper/include/clipper/task_executor.hpp

```diff
@@ -260,6 +260,21 @@ class TaskExecutor {
     }
     redis::send_cmd_no_reply<std::string>(
         redis_connection_, {"CONFIG", "SET", "notify-keyspace-events", "AKE"});
+
+    redis::subscribe_to_model_changes(redis_subscriber_,
+                    [this, task_executor_valid = active_ ](const std::string &key,
+                                                           const std::string &event_type) {
+                log_info(LOGGING_TAG_TASK_EXECUTOR,"subscribe_to_model_changes is called");
+                if (event_type == "hset" && *task_executor_valid) {
+                    auto model_info = clipper::redis::get_model_by_key(redis_connection_, key);
+                    VersionedModelId model_id = VersionedModelId(model_info["model_name"], model_info["model_version"]);
+                    int batch_size = std::stoi(model_info["batch_size"]);
+                    std::cout<<"registered_batch_size"<<batch_size<<std::endl;
```

Comment:
Replace this with `log_info_formatted("Registered batch size of {} for model {}:{}", batch_size, model_id.get_name(), model_id.get_id())`